ConorMFinn/CO600-SeriousCyberSec
# Created minimap

**Description of work.**

Created a minimap asset to the game to show the player the mapping of the house.

**To test:**

Start the game and look for the map.
Check that it correctly shows which room you're in.

Fixes #74
